pages/index.tsx

```diff
@@ -139,6 +141,13 @@ const Home: FC<IHomeProps> = ({ latestPosts, network, networkSocialsData }) => {
 	// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, []);
 
+	useEffect(() => {
```

Comment:
do we need this init here, when we already have on in the ChatFloatingModal.tsx file ?